src/GHC/Check.hs

```diff
@@ -116,15 +124,26 @@ checkGhcVersion compileTimeVersions runTimeLibdir = do
             $ do
               runTimeVersions <- collectPackageVersions (map fst compileTimeVersions)
               let compares =
-                    Map.intersectionWith
-                      comparePackageVersions
-                      compileTimeVersionsMap
-                      (Map.fromList runTimeVersions)
+                    if isUnstableGhcVersion (lookup "ghc" runTimeVersions)
+                      then Map.empty
```

Comment:
This will lead to a `PackageCheckInconclusive` verdict. Is that what you want?

Reply:
Hm, not entirely sure what I want here semantically. The compile-time version will differ from the run-time version, so it is a `VersionMismatch` but we dont care about it. The result must not be a `PackageCheckFailure` but it could be a `PackageCheckSuccess` containing a `VersionMismatch`? Would that make sense somehow?

Otherwise, I think `PackageCheckInconclusive` could actually work. That, or we introduce another state, such as `PackageCheckUnstableGhcVersion`.

Reply:
Inconclusive is interpreted by ghcide as a warning, which is probably what you want. 
A `PackageCheckSuccess` containing a `VersionMismatch` might be more precise yes, but also more surprising, potentially leading to a crash if the client didn't consider that possibility. 
A new constructor is rather overkill